vm/systemSmartContracts/liquidStaking.go

```diff
@@ -0,0 +1,244 @@
+//go:generate protoc -I=proto -I=$GOPATH/src -I=$GOPATH/src/github.com/ElrondNetwork/protobuf/protobuf  --gogoslick_out=. liquidStaking.proto
+package systemSmartContracts
+
+import (
+	"bytes"
+	"fmt"
+	"sync"
+
+	"github.com/ElrondNetwork/elrond-go-core/core"
+	"github.com/ElrondNetwork/elrond-go-core/core/atomic"
+	"github.com/ElrondNetwork/elrond-go-core/core/check"
+	"github.com/ElrondNetwork/elrond-go-core/hashing"
+	"github.com/ElrondNetwork/elrond-go-core/marshal"
+	"github.com/ElrondNetwork/elrond-go/config"
+	"github.com/ElrondNetwork/elrond-go/vm"
+	vmcommon "github.com/ElrondNetwork/elrond-vm-common"
+)
+
+const tokenIDKey = "tokenID"
+const noncePrefix = "n"
+
+type liquidStaking struct {
```

Comment:
is this sc complete? What it does is that it just have some checks on some operations. 
\+ do not forget about tests.

Reply:
No. this is incomplete.

Reply:
ok